guacamole-docker/bin/start.sh

```diff
@@ -762,6 +822,11 @@ fi
 set_property "guacd-hostname" "$GUACD_HOSTNAME"
 set_property "guacd-port"     "$GUACD_PORT"
 
+# Set up Tomcat RemoteIPValve
+if [ -n "$GUACAMOLE_PROXY_ALLOWED_IPS_REGEX" ]; then
```

Comment:
Let's mimic the `TOTP_ENABLED` option :
```suggestion
if [ "$REMOTE_IP_VALVE_ENABLED" = "true" ]; then
```
So that we are not enforced to overwrite the default Tomcat value.